### PR TITLE
Added docs for containerizing strapi

### PR DESCRIPTION
**Pull request by cotyhamilton**
#### Description of what you did:

Created docs for containerizing Strapi application

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [x] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

**Comment by lauriejim** [comment]
Hello @cotyhamilton !
Thank you for this guide, it's super good!

I have a question, is there a reason why you didn't use the `strapi/strapi` and `strapi/base` images?

Here is some `docker-compose` configs that I did for the next structure version of the docs - https://raw.githubusercontent.com/strapi/strapi/docs-v2/docs/3.0.0-beta.x/installation/docker.md

**Comment by cotyhamilton** [comment]
I did look them over but I’ve never used them

1. Development of the docker image dropped off a few months ago and just picked back up so wasn’t sure about reliability here

2. I was kind of put off by the the env variable declared in the base docker image, I know it can be overridden in the run command and docker-compose but you generally don’t bake configuration into images (although I do understand why you’d want to set a default for production)

Let me look it over and I’ll have an update soon, someone else pointed out that I was using an older Postgres image in my docker compose file, I’ll fix that too

Any other tips or considerations you have for me?

File: docs/3.0.0-beta.x/guides/docker.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+```bash
+docker build -t dockerhubaccount/project:latest .
+
+docker push dockerhubaccount/project:latest
+```
+
+You can run a single container locally with the following.
+
+```bash
+docker run -it -e NODE_ENV=development -p 1337:1337 dockerhubaccount/project
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/guides/docker.md]: Running `yarn start` with `NODE_ENV=development` won't allow users to modify the content types. You should should how to run yarn develop with the command option of docker run

**Comment by alexandrebodin** [comment]
Hi, I'm going to close this PR as now changes where made in the last 20 days. Please re-open when you are available ;)

**Pull request closed without merging**